src/js/button.js

```diff
@@ -26,24 +26,14 @@ class Button extends ClickableComponent {
    * @method createEl
    */
   createEl(tag = 'button', props = {}, attributes = {}) {
+    if (tag !== 'button') {
+      throw new Error('Buttons must be created with a "button" tag.');
```

Comment:
Well, we definitely want to accept `props` and `attributes`, I think, for custom buttons. But I suppose changing the signature isn't the worst thing in the world.